src/python/grpcio_tests/setup.py

```diff
@@ -52,6 +52,7 @@
     'preprocess': commands.GatherProto,
     'build_package_protos': grpc_tools.command.BuildPackageProtos,
     'build_py': commands.BuildPy,
+    'run_fork': commands.RunFork,
```

Comment:
Can you add this to the Python test suite?

Reply:
I just saw your comment about adding this later.

It seems like src/python/grpcio_tests/tests/fork/methods.py could easily be framed in terms of ```unittest.TestCase``` and just added to the test suite.  Is this complicated because it can only be run on certain platforms?